the-blue-alliance-ios/Core Data/TBA.xcdatamodeld/TBA.xcdatamodel/contents

```diff
@@ -77,6 +78,19 @@
         <relationship name="event" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="Event" inverseName="points" inverseEntity="Event" syncable="YES"/>
         <relationship name="team" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="Team" inverseName="eventPoints" inverseEntity="Team" syncable="YES"/>
     </entity>
+    <entity name="EventRanking" representedClassName="EventRanking" syncable="YES" codeGenerationType="class">
+        <attribute name="dq" optional="YES" attributeType="Integer 16" usesScalarValueType="NO" syncable="YES"/>
+        <attribute name="extraStats" optional="YES" attributeType="Transformable" syncable="YES"/>
```

Comment:
For these transformables... we can add some type safety around them. Check out what we do in `Event` for `divisionKeys` and `insights` https://github.com/the-blue-alliance/the-blue-alliance-ios/blob/swift/the-blue-alliance-ios/Core%20Data/Event.swift#L27

You have to change the `Custom Class` field in the property editor in Core Data but that's pretty easy
https://github.com/the-blue-alliance/the-blue-alliance-ios/blob/swift/the-blue-alliance-ios/Core%20Data/TBA.xcdatamodeld/TBA.xcdatamodel/contents#L44